### ISSUE TITLE
Accessibility insights report on pagination colour contrast in XFP

### ISSUE THREAD

**Issue by markbjadu** [jadu/pulsar#983]
Pagination Elements must have sufficient colour contrast

1. Go to 'Xforms Pro > and select or create a new form'
2. Click on 'Advanced tab', create and save some rules. 
3. Go back to advanced tab and run 'Accessibility insights'
4. See the report

"Element has insufficient color contrast of 1.85 (foreground color: #bebebe, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1"

 - OS: [mac OS 10.14.5]
 - Browser [chrome]
 - Version [74.0.3729.157]

![Screenshot 2019-05-21 at 10 11 45](https://user-images.githubusercontent.com/6906173/58083629-447d4900-7bb1-11e9-882d-b949b951f772.png)

**Comment by markbjadu** [comment]
Shatabdi mentioned that the forwards and previous only need to be visible if there are extra pages. Can they be hidden if there is just one page the << >>? @Stanton.

**Comment by Stanton** [comment]
Basic tables will need their pagination routines updating in-product to match the update recommendations. Keeping ticket open for the time being.

**Comment by Stanton** [comment]
No longer an issue after Purple
